Source/SwiftLintFramework/Rules/Lint/HashValueOverflowRule.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+import SourceKittenFramework
+
+public struct HashValueOverflowRule: ASTRule, OptInRule, ConfigurationProviderRule, AutomaticTestableRule {
```

Comment:
IMO this doesn't need to be an opt-in rule

Reply:
Sounds good to me! 
Will remove it